src/language-server/config/config.ts

```diff
@@ -104,6 +106,21 @@ const roverConfig = z.object({
       },
     ),
   profile: z.string().optional(),
+  supergraphConfig: z
+    .preprocess((value) => {
+      if (value !== undefined) return value;
+      const configPath = contextStore.getStore()?.configPath!;
+      const supergraphConfig = join(configPath, "supergraph.yml");
+      return existsSync(supergraphConfig) ? supergraphConfig : undefined;
+    }, z.string().nullable().optional())
+    .describe(
+      "The path to your `supergraph.yml` file. \n" +
+        "Defaults to a `supergraph.yml` in the folder of your `apollo.config.js`, if there is one.",
+    ),
+  extraArgs: z
+    .array(z.string())
+    .default([])
+    .describe("Extra arguments to pass to the Rover CLI."),
```

Comment:
I believe these should be enough for now, and `extraArgs` opens the door for anything I might have forgotten for now :)